### ISSUE TITLE
Establish criteria to remove flipper facility_locator_ppms_forced_unique_id                            

### ISSUE THREAD

**Issue by dsinla** [department-of-veterans-affairs/va.gov-cms#11593]
## Description
As we investigate each Facilities related Flipper, document the criteria and timeline for when it should be removed.

Refer to https://github.com/department-of-veterans-affairs/va.gov-cms/issues/10656

## Acceptance Criteria
- [ ] Criteria for Flipper removal is created in this ticket 

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide Crew`
- [ ] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public Websites`
- [X] `⭐️ Facilities`
- [ ] `⭐️ User support`

**Comment by maxx1128** [comment]
I checked `content-build` and `vets-website` and found no reference to this flipper in any code files. It's already been removed from the front end, so the only step left is to remove the toggle from `vets-api` which is an easy change.

**Comment by jilladams** [comment]
@maxx1128 just saw your Slack DMs. One note here -- we found previously that the name of the flipper as snake case won't always appear in vets-website code. Instead, [notes here](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/10342#issuecomment-1324147858) on how it gets translated to camelCase. That camelCase reference is really what we need to confirm doesn't exist anywhere in the front end.

Can you take another look with that linked comment in mind and confirm if this flipper is safe to remove. (cc @ryguyk )

**Comment by jilladams** [comment]
Also noting, this flipper is currently enabled in production: https://api.va.gov/flipper/features

**Comment by maxx1128** [comment]
I double-checked and there are still no references to this flipper in either of the front-end codebases.

**Comment by maxx1128** [comment]
The branches for removing the flipper from the needed repos are below. No pull requests have been made with them yet, but can be once this flipper is approved for removal.

1. `vets-api` - `va-11593-remove-ppms-forced-unique-id-feature`

**Comment by jilladams** [comment]
@maxx1128 that in mind: can this ticket be closed?

**Comment by maxx1128** [comment]
@jilladams Yes I think it can be closed as is since it's having zero effect on the front-end from what I can see.
